controller/array_action/array_mediator_xiv.py

```diff
@@ -76,17 +84,26 @@ def disconnect(self):
         if self.client and self.client.is_connected():
             self.client.close()
 
-    def _convert_size_blocks_to_bytes(self, size_in_blocks):
-        return size_in_blocks * self.BLOCK_SIZE_IN_BYTES
+    def _get_volume_or_snapshot_size_in_bytes(self, volume_or_snapshot):
```

Comment:
I didn't change. I added another method which uses this one.
Do u have a better suggestion for name? We can't call it "object" since it is specific for volume and snapshot

Reply:
not sure which method you added (looks like a rename in this diff).
I'm still not sure why can't just we keep the `_convert_size_blocks_to_bytes` with the `size_in_blocks` parameter.
is it so bad for the `_generate_*_response` functions to access `.capacity`?

Reply:
left old method name